src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Repeater/Given_ItemsRepeater.cs

```diff
@@ -43,11 +47,12 @@ public async Task When_NoScrollViewer_Then_ShowMoreThanFirstItem()
 			sut.UpdateLayout();
 
 			var second = sut
-				.EnumerateAllChildren()
+				.GetAllChildren()
```

Comment:
Why ?

Reply:
Because `EnumerateAllChildren` is not available on UWP while `GetAllChildren` is